targets/targets.json

```diff
@@ -5270,6 +5270,31 @@
             "target.network-default-interface-type": "ETHERNET"
         }
     },
+    "S5JS100": {
+        "inherits": ["Target"],
+        "core": "Cortex-M7",
+        "components_add": ["FLASHIAP"],
+        "supported_toolchains": ["GCC_ARM", "IAR", "ARM"],
+        "default_toolchain": "GCC_ARM",
+        "extra_labels": ["Samsung", "SIDK_S5JS100"],
+        "macros": ["CMSDK_CM7", "MBED_MPU_CUSTOM"],
+        "device_has": [
+            "ANALOGIN",
+            "I2C",
+            "INTERRUPTIN",
+            "PORTIN",
+            "PORTINOUT",
+            "PORTOUT",
+            "SERIAL",
+            "SPI",
+            "SLEEP",
+            "FLASH",
+            "STORAGE"
+        ],
+        "features": ["STORAGE", "SSS"],
```

Comment:
Please remove "features": ["STORAGE". Not needed and deprecated

Reply:
Hi, @dannybenor. Thanks for the feedback. I just submitted 82a2a476a30891d0854712d785ee08c1c37e64f9.